src/backends/pytorch/libtorch_backend.cc

```diff
@@ -50,6 +51,10 @@ LibTorchBackend::Context::Context(
 
 LibTorchBackend::Context::~Context()
 {
+  torch_model_.reset();
+  c10::cuda::CUDACachingAllocator::emptyCache();
+  input_index_map_.clear();
+  output_index_map_.clear();
```

Comment:
I don't think you need to call `clear()` explicitly. The same for `torch_model_` unless it needs to be destroyed before calling `emptyCache()`?

Reply:
Yes the clear is not needed. No, it isn't necessary to reset the `torch_model_` before calling `emptyCache()`.